### ISSUE TITLE
tileset delete command missing

### ISSUE THREAD

**Issue by rclark** [mapbox/tilesets-cli#63]
Hi! I noticed this today:

```
❯ tilesets delete rclark.grand-canyon-geology-v2
Usage: tilesets [OPTIONS] COMMAND [ARGS]...
Try 'tilesets --help' for help.

Error: No such command 'delete'.
```

Also noticed that the help text doesn't know that there should be a delete command.

```
❯ tilesets --help
Usage: tilesets [OPTIONS] COMMAND [ARGS]...

  This is the command line interface for the Mapbox Tilesets API. Thanks for
  joining us.

  This CLI requires a Mapbox access token. You can either set it in your
  environment as "MAPBOX_ACCESS_TOKEN" or "MapboxAccessToken" or pass it to
  each command with the --token flag.

Options:
  --version  Show the version and exit.
  --help     Show this message and exit.

Commands:
  add-source       Create/add a tileset source tilesets add-source...
  create           Create a new tileset with a recipe.
  delete-source    Delete a Tileset Source + all of its files.
  job              View a single job for a particular tileset.
  jobs             View all jobs for a particular tileset.
  list             List all tilesets for an account.
  list-sources     List all Tileset Sources for an account.
  publish          Publish your tileset.
  status           View the current queue/processing/complete status of
                   your...

  update-recipe    Update a Recipe JSON document for a particular tileset...
  validate-recipe  Validate a Recipe JSON document tilesets validate-recipe...
  validate-source  Validate your source file.
  view-recipe      View a tileset's recipe JSON tilesets view-recipe...
  view-source      View a Tileset Source's information tilesets view-source...
```

Just in case it means anything, here's my python version:

```
❯ python --version
Python 3.7.7
```

**Comment by mapsam** [comment]
Hey @rclark! The `delete` command was _just_ added in #61 and is [not released yet](https://github.com/mapbox/tilesets-cli/blob/master/CHANGELOG.md#unreleased). We have a couple more commands to work on before we release. 

Apologies that the master docs are out of sync with the latest version!

**Comment by dianeschulze** [comment]
Thanks @rclark . This is resolved with the release of v1.1.0.dev0
